addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -601,7 +601,7 @@
 			<control type="image">
 				<left>0</left>
 				<top>80</top>
-				<width>920</width>
+				<width>970</width>
```

Comment:
Was this meant to be changed because the dialog background values and focused layout haven't been?

Reply:
Change that should have been reverted, fixed